backend/src/test/java/ca/bc/gov/hlth/mohums/integration/MoHUmsIntegrationTests.java

```diff
@@ -121,6 +121,26 @@ public void addUser() throws Exception {
         // We expect a 409 (Conflict) because the user already exists.
     }
 
+    @Test
+    public void assignedUserClientRoleMappingUnauthorized() throws Exception {
+        webTestClient
+                .get()
+                .uri("users/39f73cbd-dbf0-41c6-a45c-997c44c1c952/role-mappings/clients/1b2ce61a-1235-4a0e-8334-1ac557151757")
```

Comment:
I should add a comment to explain what client this is.

Reply:
agreed